src/Microsoft.SqlTools.ServiceLayer/SqlProjects/Contracts/SqlObjectScripts/MoveSqlObjectScript.cs

```diff
@@ -0,0 +1,28 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#nullable disable
```

Comment:
for simple files like this you may consider making them nullable-compliant when adding them to avoid accumulating debt in this area.

Reply:
This was intentional; I elaborated on why over in this comment in another PR: https://github.com/microsoft/sqltoolsservice/pull/1858/files#r1102060833

Reply:
Sounds good, as long as you're making a deliberate choice here.  I assume there will likely always be some code in this project where we'll keep nullable disabled for various reasons.